xml/System.Diagnostics/Activity.xml

```diff
@@ -224,8 +224,13 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>This is an ID that is specific to a particular request.
+Filtering to a particular ID ensures that you get only one request that matches.
+An ID has a hierarchical structure: '|root-id.id1_id2.id3_'.
+The ID is generated when <see cref="M:System.Diagnostics.Activity.Start" /> is called by appending suffix to Parent.Id or <see cref="P:System.Diagnostics.Activity.ParentId" />.
+An <see cref="T:System.Diagnostics.Activity" /> has no ID until it starts.
+See <see href="https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md#id-format" /> for more details.</summary>
```

Comment:
@wtgodbe can you please coordinate with Maira to add this information in our Docs site instead of Github?

Reply:
@mairaw is there somewhere on the docs site where we can host a descriptive user guide like the one in https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/ActivityUserGuide.md? The information there doesn't seem well-suited for porting into these xml docs, but it's also a very useful document so it'd be a shame to remove the reference to it.

Reply:
Yes @wtgodbe. Our descriptive guidance lives in this other repo:  https://github.com/dotnet/docs/

It uses Markdown as well. First thing would be to define where in the TOC this document would live and it would need to be cleaned to be just a usage guide and not an architecture document.